### ISSUE TITLE
Suddenly removing all preflight

### ISSUE THREAD

**Issue by StephixOne** [Roman86/tailwindcss-scoped-preflight#56]
**Describe the bug**
Started hitting a bug in prod, noticing weird unexpected styles (buttons having borders, for example). Doing some debugging at the moment, and it looks like if I specify plugins like so:

```
plugins: [
    require("tailwindcss-animate"),
    scopedPreflightStyles({ isolationStrategy: isolateInsideOfContainer(".wrapper", { remove: ["h1", "h2", "h3", "h4", "h5", "h6"] }) })
  ]
```

or even making the usage simpler:
```
    scopedPreflightStyles({ isolationStrategy: isolateInsideOfContainer(".wrapper") })
```

the resulting CSS bundle is missing preflight completely.

You can see in the produced bundle that the comments for preflight are there, but absolutely no classes:
![image](https://github.com/user-attachments/assets/72bdcc27-81d3-41cb-a03b-e3d9ec0d6d41)

We haven't changed anything in versions or the config as far as I can tell, but it started breaking last week.

**To Reproduce**
Wish I knew 

**Expected behaviour**
The preflight styles are added and scoped under `.wrapper`, and `h1-6` preflight styles are removed.

**Additional context**
Oddly I can't reproduce locally, have been debugging in the codebuild container that runs this process.

**Comment by r4m-roman** [comment]
Hi @StephixOne ! If you hasn't upgraded the any packages and suddenly started to break - then it's somewhere on your side. Try different/older versions - should be the same. As an advice - make sure to have your ".wrapper" element in your templates code. If it's not found - tailwind styles won't appear in the output. It's not about my plugin, it's about how tailwind and/or postcss works afaik: only classnames found in the codebase your compile are getting into your final CSS. Maybe there is some externally/dynamically imported (during the runtime) code - it can't appear in your CSS during the build time.

**Comment by StephixOne** [comment]
Okay I think I know what's happened, sort of. Something else is buggy but it's unrelated to the plugin. But essentially we have specific page layouts that are wrapped in and import a generic master layout, which is what has that `wrapper` class. The path to the specific layouts is what's being fed to the tailwind process, but for some reason tailwind has stopped picking up classes from the imported master layout. 

I've given it the path to the master layout and now things are being scoped properly again 🙏. I had to fix another issue a couple of weeks back that was caused by a similar thing, so I should've thought about it, I'd just forgotten about this dependency structure.

Thanks for the help and for the plugin!
